python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```diff
@@ -1168,6 +1168,9 @@ $defs:
                     timers:
                       type: str
                       description: BGP Keepalive and Hold Timer values in seconds as string <0-3600> <0-3600>.
+                    bfd_timers:
+                      type: dict
+                      $ref: "eos_cli_config_gen#/keys/router_bgp/keys/vrfs/items/keys/neighbors/items/keys/bfd_timers"
```

Comment:
We can add a description specifying the requirements for using bfd_timers, including any recommendations similar to those provided for other bfd_timers keys.

Reply:
The DPS stuff does not apply to these peers, so I don't think we should have that